README.md

```diff
@@ -86,6 +86,37 @@ It is manadatory to load an authorization JSON file, if no authorization file is
 
 The keycloak event-emitter module sends all events to the bridge's event endpoint. The event emitter use HTTP with flatbuffers.
 
+### Monitoring of keycloak-bridge
+
+An endpoint allows to get a status of the Bridge and its components health.
+URL: ```http://{bridge.host}:{bridge.port}/health/check```
+
+Status example:
+```
+{
+ "Audit R/W": {
```

Comment:
Maybe add something to state it is database connection ?

Reply:
Something like this ?
```
{
 "Audit R/W": {
  "state": "UP",
  "type": "database",
  "connection": "established"
 },
 "Keycloak": {
  "state": "DOWN",
  "type": "http",
  "message": "Can't hit target: Get http://127.0.0.1:8080: dial tcp 127.0.0.1:8080: connectex: No connection could be made because the target machine actively refused it."
 }
```